app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1><%= @dish.name %></h1>
+<p><%= @dish.description %></p>
+
+<h3>Ingredients</h3>
+<div id="ingredients">
+   <ul>
+   <% @dish.ingredients.each do |ingredient| %>
+      <div id="ingredient_<%= ingredient.id %>">
+      <li><%= ingredient.name %></li>
+      <%= form_with model: @dish, url: destroy_ingredient_dish_path(@dish), method: :delete do |form| %>
```

Comment:
form_with model is just another way to set up your forms, but I generally think it's best only when you're using it to create or edit a whole object, rather than doing something like creating a joins record. It can sometimes complicate things more for forms where you're doing something different (like in this case) so just be careful! You don't *have* to use it, it's just another tool in your toolbox